test/crypto_sign/functest.c

```diff
@@ -182,8 +199,12 @@ static int test_sign(void) {
 
         // By relying on m == sm we prevent having to allocate CRYPTO_BYTES
         // twice
-        if ((returncode =
-                    crypto_sign_open(sm + 8, &mlen, sm + 8, smlen, pk + 8)) != 0) {
+        #ifdef PQCLEAN_USE_SIGN_CTX
+        returncode = crypto_sign_open(sm + 8, &mlen, sm + 8, smlen, ctx + 8, PQCLEAN_CTXLEN, pk + 8);
```

Comment:
We should also add a test in which the context is not set/set differently, and verification fails.